src/app/admin/books/page.tsx

```
@@ -0,0 +1,53 @@
+      </Typography>
+      <Button
+        variant="contained"
+        color="primary"
+        href="/admin/books/new"
+        sx={{ mt: 2, alignSelf: "flex-start" }}
+      >
+        新しい本を登録
+      </Button>
+      <Stack spacing={2} mt={4}>
+        {books.map((book) => (
+          <Card variant="outlined">
```

Biome (2.4.4)

[error] Missing key property for this element in iterable.

(lint/correctness/useJsxKeyInIterable)